src/platform/RaribleService.ts

```diff
@@ -116,6 +116,7 @@ export class RaribleService implements IPlatformService {
       platformLogoUrl: this.platformLogoUrl,
       nftName: nftInfo.meta.name,
       nftUri: nftInfo.meta.content[0].url,
+      tokenStandard: "erc721",
```

Comment:
From the docs I think this could also be 1155 but I couldn't find field in items API